lib/src/flagsmith_client.dart

```diff
@@ -88,6 +100,63 @@ class FlagsmithClient {
     return null;
   }
 
+  Future<void> _setupRealtimeUpdates(String realtimeUpdatesBaseUrl) async {
+    final sseUrl = '$realtimeUpdatesBaseUrl$apiKey/stream';
+
+    _sseStream = SSEClient.subscribeToSSE(
+      method: SSERequestType.GET,
+      url: sseUrl,
+      header: {
+        acceptHeader: '$applicationJson , text/event-stream',
```

Comment:
Only because we were using `$applicationJson` in multiple places but `text/event-stream` only in one. I'll extract `text/event-stream` too for consistency